src/components/CreateTeam/index.jsx

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
+import { Redirect } from 'react-router-dom';
 import { PropTypes as PT } from 'prop-types';
 import container from './container';
 import CreateTeamForm from './form';
 import { Button } from '../common';
 import { CornerButton } from '../common_styled';
 
-export const CreateTeam = ({ clients, submitRequest, navigateTo }) => {
+export const CreateTeam = ({ clientOptions, submitRequest, navigateTo }) => {
+
+  const renderRedirect = () => {
```

Comment:
I've seen this `renderRedirect` in multiple files can we break it out into a helper/utility func?

Reply:
Yes, it could be when everything is merged.